src/openrct2-ui/scripting/CustomWindow.cpp

```diff
@@ -950,8 +964,21 @@ namespace OpenRCT2::Ui::Windows
                 }
             }
 
-            SetWidgets(widgetList);
+            // Fix for plugin windows not resize correctly when switching tabs
+            int16_t maxBottom = 0;
+            for (const auto& widget : widgetList)
+            {
+                if (!(widget.flags & WIDGET_FLAGS::IS_HIDDEN))
+                {
+                    maxBottom = std::max(maxBottom, widget.bottom);
+                }
+            }
+
+            // +5 just for extra padding at the bottom
+            const int16_t newHeight = maxBottom + 5;
+            height = newHeight;
```

Comment:
Could you motivate these changes? This seems like a way to determine the window height, but this really ought to be resized to automatically by `WindowBase::ResizeFrame`.